specifyweb/frontend/js_src/lib/components/Merging/InvalidMergeRecords.tsx

```diff
@@ -37,19 +37,26 @@ function InvalidMergeRecords({
             className="flex min-h-[theme(spacing.8)] flex-1 items-center gap-2"
           >
             <TableIcon label={true} name={tableName} />
-            <FormattedResource
-              resource={React.useMemo(
-                () => deserializeResource(resource),
-                [resource]
-              )}
-            />
+            <FormattedMemoizedResource resource={resource} />
           </li>
         ))}
       </Ul>
     </div>
   );
 }
 
+function FormattedMemoizedResource({
```

Comment:
what I meant is for the entire contents of the .map() to be turned into a component - that way the component will serve a bit more purpose than just useMemo

and then you would be able to rename `FormattedMemoizedResource` to just `Record` or `InvalidMergeRecord` (using record instead of resource because that's what you used in `InvalidMergeRecords` - consistency 🤵‍♂️)

please refactor - should be a simpler change

Reply:
Isn't that what it is right now?

Reply:
All the contents of .map is a new component (https://github.com/specify/specify7/pull/3832/commits/9be6ce28a5e97585627c527c0e7f93465dd8a302)

Reply:
you are right. looks like I was reviewing outdated code somehow